build.rs

```diff
@@ -89,12 +90,21 @@ fn windows_compile_tsk() {
     let msbuild_path = install_path.join(r"MSBuild\Current\Bin\MSBuild.exe");
     eprintln!("msbuild_path -> {:?}\n", msbuild_path);
 
+    // Fix libtsk.vcxproj (can be removed once https://github.com/sleuthkit/sleuthkit/pull/2205 is merged and released upstream)
+    let libtsk_vcxproj_contents = fs::read_to_string(r"sleuthkit\win32\libtsk\libtsk.vcxproj").unwrap();
+    fs::write(r"sleuthkit\win32\libtsk\libtsk.vcxproj",
+              libtsk_vcxproj_contents.replace(r#"<Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">"#,
+                                              r#"<Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild" Condition="!$(Configuration.EndsWith('_NoLibs'))">"#))
+              .unwrap();
+
+
     let output = Command::new(&msbuild_path)
         .args(&[
             r"-target:libtsk",
             r"/p:PlatformToolset=v142",
             r"/p:Platform=x64",
             r"/p:Configuration=Release_NoLibs",
+            r"/p:RestorePackages=false",
```

Comment:
Can you explain this one to me? I am trying to learn more about the building process.

Reply:
This tells MSBuild not to restore NuGet packages (since we are compiling with NoLibs we don't need them)